index.js

```diff
@@ -2,3 +2,5 @@ exports.client = require('./client');
 exports.cluster = require('./cluster');
 exports.process = require('./process');
 exports.server = require('./server');
+exports.ws_channel = require('./ws-channel');
```

Comment:
I mean, I pushed the lint configuration to master, you'll get it when you rebase.